problem3.py

```diff
@@ -0,0 +1,59 @@
+"""
+Created on Fri Mar 16 23:27:24 2018
+@author: patricio
+
+For Solving Problem 3 in Project Euler.
+
+Three functions were defined:
+
+1) is_prime: Receives a number and determines if the number is prime.
+    
+2) find_next_prime_num_after: Returns the next prime number that follows
+    the imputed number
+    
+3) find_largest_prime_factor: This function calculates all prime factors
+    by starting at the smallest prime and working upwards. Then it returns
+    the maximum of the list.
+"""
+
+def is_prime(number):
+    default = True
+    if number in [0,1]:
+        default = False
+    elif number == 2:
+        default = True
+    else:
+        for i in range(2,number//2+1):
+            if number % i == 0:
+                default = False
+                break
+    return default
+
+def find_next_prime_num_after(num):
+    condition_meet = False
+    if num % 2 == 0:
+        k = num//2 #Since we're only checking numbers of the form 2k+1
```

Comment:
Do you need to look this high?

Reply:
A different algorithm will be implemented, but no I don't need to look this high... lol